content/docs/tasks/traffic-management/egress/index.md

```diff
@@ -264,6 +264,102 @@ $ export SOURCE_POD=$(kubectl get pod -l app=sleep -o jsonpath={.items..metadata
 $ kubectl exec -it $SOURCE_POD -c sleep curl http://httpbin.org/headers
 {{< /text >}}
 
+## Install Istio with access to all external services by default
+
+An alternative to calling external services directly is to instruct the Istio proxy to pass through the calls to all the
+external services. This option allows you to start evaluating Istio quickly, without controlling access to external
+services, and decide to [configure access to external services](#configuring-istio-external-services) later.
+
+Istio has an installation option that allows access to all external services for which no HTTP service exists inside
+the mesh and no HTTP/TCP `ServiceEntry` is defined. For example, if your mesh does not have an HTTP service on port 443,
+and you did not define a `ServiceEntry` on port 443, you can access any external service on port 443. Note, however,
+that once you create an HTTP service on port 443 or define any `ServiceEntry` for any host on
+port 443, all accesses to port 443 is blocked: Istio will fall back to the blocking-by-default behavior for that
+port only. (Defining an HTTP service on port 443 is not recommended anyway, since using the same port for TCP/HTTPS and
+for HTTP traffic in Istio is discouraged.)
```

Comment:
again, should we move this to a troubleshooting section to avoid confusion?  (since this is not recommended or common to have http on port 443).

Reply:
Addressed in [c841ed4](https://github.com/istio/istio.io/commit/c841ed4f438123acf09e3fec3c08c4e5e30578e5). I use  port 8000 instead 443 and remove the sentence in parenthesis.